es6.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  extends: ['eslint-config-mm/es5'],
+  parserOptions: {
+    ecmaVersion: 6
+    // Can't use ES6 imports yet in node: https://github.com/nodejs/help/issues/53#issuecomment-165763446
+    // sourceType: 'module'
+  },
+  env: {
+    node: true,
+    es6: true
+  },
+  rules: {
+    'array-callback-return': 1,
+    'arrow-parens': [1, 'always'],
+    'arrow-body-style': [1, 'always'],
+    'arrow-spacing': 1,
+    'no-const-assign': 1,
+    'no-iterator': 1,
+    'no-var': 1,
+    'object-shorthand': 1,
+    'prefer-arrow-callback': 1,
+    'prefer-const': 1,
```

Comment:
Does this mean eslint warns where var is used, where it can also use const? I'm not a huge fan of this because const does not imply immutability. Also it will probably complain a lot(!) about the current codebase.

Let's think this through? I'm probably easy to convince on this anyway. I just like your take on it.